src/docs/reference/modules/Markdown/README.md

```diff
@@ -166,6 +166,48 @@ To disable sanitization:
 @await Orchard.MarkdownToHtmlAsync((string)Model.ContentItem.Content.MarkdownParagraph.Content.Markdown, false)
 ```
 
+## Markdown Settings
+
+Enabling the `OrchardCore.Markdown` module will allow the user to set the following settings:
+
+| Setting | Description |
```

Comment:
The table for supported extensions looks great!

Suggest instead of the table here for the settings it would be better to just show the configuration json (for the default values). So exactly what you have added to the `appsettings.json` file without the comments.

(Otherwise from reading this, I have no idea where to put the `Extensions` setting)

Example here https://docs.orchardcore.net/en/dev/docs/reference/modules/Media/#configuration

Also I think the https://docs.orchardcore.net/en/dev/docs/reference/modules/Markdown/#markdown-pipeline
pipeline options should be shown second, after this, as this is the more convenient way to configure.

So happy for you to move the pipeline options section to after this.

Reply:
Thanks Dean for your comments and suggestion, I will update the PR ASAP